### ISSUE TITLE
Missing MIDs more than 9 in newly released strong labels

### ISSUE THREAD

**Issue by lijuncheng16** [audioset/ontology#9]
Hi Dan: 
Sorry for bugging again!
"The file [audioset_eval_strong.tsv](http://storage.googleapis.com/us_audioset/youtube_corpus/strong/audioset_eval_strong.tsv) describes 139,538 segments across the 16,996 excerpts from the evaluation set. There are 416 MIDs, 9 of which are not present in the train labels."

In my experiment, there seem to be 35 MIDS that are different than the original weakly labeled 527 MIDS: 
{'/m/0bzvm2', '/t/dd00139', '/t/dd00098', '/m/07q8f3b', '/m/0c1tlg', '/m/0md09', '/t/dd00091', '/m/093_4n', '/m/01sb50', '/m/0174k2', '/m/01j423', '/m/0hgq8df', '/t/dd00099', '/m/05mxj0q', '/t/dd00141', '/m/01lynh', '/m/0fw86', '/m/0dgw9r', '/t/dd00061', '/t/dd00109', '/m/09l8g', '/m/07sk0jz', '/t/dd00133', '/m/0d4wf', '/m/018p4k', '/t/dd00143', '/m/0bcdqg', '/m/09hlz4', '/m/0zmy2j9', '/t/dd00138', '/t/dd00142', '/m/02f9f_', '/m/02021', '/m/01j3j8', '/m/0641k'}

Just want to confirm this is the expected behavior. 

This is very similar to [Issue 6](https://github.com/audioset/ontology/issues/6), but not quite the same.

Thank you in advance!

**Comment by dpwe** [comment]
You're quoting from [AudioSet / Temporally-Strong Labels Download (May 2021)](https://research.google.com/audioset/download_strong.html):


> The file [audioset_train_strong.tsv](http://storage.googleapis.com/us_audioset/youtube_corpus/strong/audioset_train_strong.tsv) describes 934,821 sound events across the 103,463 excerpts from the training set. There are 447 MIDs present. Note that this comprises significantly more than the 66,924 excerpts promised in the ICASSP paper, reflecting additional annotations collected since writing the paper.

> The file [audioset_eval_strong.tsv](http://storage.googleapis.com/us_audioset/youtube_corpus/strong/audioset_eval_strong.tsv) describes 139,538 segments across the 16,996 excerpts from the evaluation set. There are 416 MIDs, 9 of which are not present in the train labels.

So the 9 MIDs not present in `audioset_eval_strong.tsv` refers to differences from the first file, `audioset_train_strong.tsv`.

Of the 447 MIDs in `audioset_train_strong.tsv`, 376 are present in the original AudioSet weak label release, and 71 are not.  These 71 account for all but 3 of the 35 MIDs you identify as present in `audioset_eval_strong.tsv` but not in the original AudioSet weak labels.  Of the 9 MIDs we mention as present in strong-eval but not in strong-train, 6 are part of the weak-label set, and 3 are not.  Of the 40 labels present in strong-train but not in strong-eval, only one ("Zing") is from the original weak labels.  So there are 375 MIDs common to strong-train, strong-eval, and the original weak-labels.

The overall label overlap can be summarized:

| | weak | strong train | strong eval|
|-|-|-|-|
|weak| 527| 376| 381|
|strong train | | 447| 407|
|strong eval | | | 416|

It was important not to drop labels from the strong-labeling dataset in order to preserve the "complete labeling" property of the data (i.e., that every perceptible event was labeled).  And we chose not to attempt to project them onto some smaller subset in order to preserve as much information as possible.  But these differing label subsets are the result.

I regret that we weren't more explicit about the overlap in MIDs with the original weak data release, I'll update the page.

**Comment by lijuncheng16** [comment]
Hi Dan: 
Thank you so much for the clarification! This is very helpful!
